README.md

```diff
@@ -89,6 +89,10 @@ Every week, we meet to plan and sync on the progress for the milestone's release
 **Password:** Ask @maintainers in our Discord server<br>
 **Schedule:** Tuesdays at 9am and 9pm Pacific Time.
 
+### JavaScript SDK sync meetings
+
+Every week, we meet to plan and sync on the progress for the milestone's release. You can find more information about it here: [Community engagement - dapr/js-sdk Wiki](https://github.com/dapr/js-sdk/wiki/Community-engagement)
+
```

Comment:
```suggestion
#### JavaScript SDK Milestone Sync 

Every week, the JavaScript maintainers meet to plan and sync on the progress for the SDK's milestone release. These meetings are open to the entire Dapr community. More information about the weekly JavaScript sync can be found here: [Community engagement - dapr/js-sdk Wiki](https://github.com/dapr/js-sdk/wiki/Community-engagement)

```

Reply:
Let's clarify that these calls are hosted by JavaScript maintainers and open to the entire dapr community.

Reply:
Yes, that is correct.